src/processor_x86.cpp

```diff
@@ -1084,16 +1082,14 @@ jl_image_t jl_init_processor_pkgimg(void *hdl)
 {
     if (jit_targets.empty())
         jl_error("JIT targets not initialized");
-    if (jit_targets.size() > 1)
```

Comment:
I think we need to test that we still pick the right/one for all.

I have a unsubstantiated fear about loading images with mismatched ABI. Previously this code said to pick one and then ensured that everyone is consistent.